BlazarBase/src/main/java/com/hubspot/blazar/base/BuildCommand.java

```diff
@@ -0,0 +1,61 @@
+package com.hubspot.blazar.base;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+import com.google.common.collect.Lists;
+
+public class BuildCommand {
+  private static String DEFAULT_CMD = "/bin/bash";
+  private static String DEFAULT_ARG = "-c";
+
+  private final String executable;
+  private final List<String> args;
+  private final List<Integer> returnCodes;
+  private final Map<String, String> commandSpecificEnvironment;
```

Comment:
It seems verbose here, but I'd like to keep it clearly separate from the other environments that we have floating around, such as the environment for this section of the build. And the environment for the whole build.